db/view/view.cc

```diff
@@ -188,6 +188,16 @@ db::view::base_info_ptr view_info::make_base_dependent_view_info(const schema& b
     std::vector<column_id> base_regular_columns_in_view_pk;
     std::vector<column_id> base_static_columns_in_view_pk;
 
+    _is_partition_key_permutation_of_base_partition_key =
```

Comment:
Please use the word "Fixes". not "closes", in the commit message. Although github treats both the same, our own automation doesn't.